### ISSUE TITLE
Warning: Using the last argument as keyword parameters is deprecated

### ISSUE THREAD

**Issue by rinaldifonseca** [dry-rb/dry-transaction#132]
### Steps to reproduce
- Install ruby 2.7.1
- Use dry-transaction

### Expected behavior
- No warning or deprecation messages

### Actual behavior
Warning:
/Users/jow/.asdf/installs/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/dry-transaction-0.13.0/lib/dry/transaction/instance_methods.rb:17: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
/Users/jow/.asdf/installs/ruby/2.7.1/lib/ruby/gems/2.7.0/gems/dry-transaction-0.13.0/lib/dry/transaction/instance_methods.rb:84: warning: The called method `resolve_operation' is defined here

I found the post below and using the hash splat(**operations) in the line 17 of lib/dry/transaction/instance_methods.rb
fixed the warning. Is this the path to follow?

https://piechowski.io/post/last-arg-keyword-deprecated-ruby-2-7/

**Comment by milushov** [comment]
I have also this warning:
```
/Users/fuck/.rvm/gems/ruby-2.7.1/gems/dry-transaction-0.13.0/lib/dry/transaction/instance_methods.rb:84: warning: The called method `resolve_operation' is defined here
```

**Comment by trammel** [comment]
This is now a breaking problem on the released ruby 3.0.0

**Comment by colszowka** [comment]
Note there seems to be a PR that fixes the deprecations at https://github.com/dry-rb/dry-transaction/pull/134. Would be very nice to get this resolved, thank you!

**Comment by timriley** [comment]
I'll take a look at this in the next day or two, thanks folks.

**Comment by timriley** [comment]
This is fixed via #134 and released in 0.13.1
